### ISSUE TITLE
empty page with docker use

### ISSUE THREAD

**Issue by tierline1** [caioricciuti/ch-ui#39]
see screenshot
![Снимок экрана 2024-11-14 145406](https://github.com/user-attachments/assets/8a75babb-77c3-40f0-8361-0c7ed1167135)
![Снимок экрана 2024-11-14 145329](https://github.com/user-attachments/assets/c1022282-5f06-4103-b8cc-5fbac8c9e2d8)

**Comment by caioricciuti** [comment]
Hey, are you able to run the CH-UI if you don't set the docker variables?

Also, please check issue #8.

I think this is not an issue with CH-UI rather than the connection with click house itself, but I'll check, it should run and show an error if that's the case..

thanks!

**Comment by caioricciuti** [comment]
@tierline1 could you make it work?

**Comment by caioricciuti** [comment]
Closing the issue since no feedback, please reopen if needed.

**Comment by mwarnerdotme** [comment]
I'm having this same issue actually. This container has been working fine for me for several days, and then suddenly it just seems to 304 to a blank page. I was supplying the url/username/password via env vars, and then this started happening. I tried removing all of the env vars, and I still get the same result. I've even tried deleting the image and pulling it again, and that also had no effect.

<img width="746" alt="image" src="https://github.com/user-attachments/assets/86ed07c0-1aed-4c26-9399-69fa9f485b7f">

**Comment by mwarnerdotme** [comment]
I was just able to clear the "white screen issue" by clearing local storage. Looks like an invalid URL (I tried using the clickhouse:// protocol) was still being stored in local storage despite having reset the server. For some reason this is causing a 304.

tl;dr: clear your local storage to reset. `ctrl+shift+r` alone won't work.

**Comment by caioricciuti** [comment]
@mwarnerdotme very good catch, I didn't took that into consideration... I'll figure it out a way of cleaning the storage if the connection fails or times out! Niceee!

**Comment by caioricciuti** [comment]
It's fixed on today's release
